templates/vue/src/components/Lightbox/media/VideoMedia/UrlVideoMedia.vue

```diff
@@ -147,6 +150,24 @@ export default {
         currentTime,
       })
     },
+    async handleError() {
+      // Error code 4 - The associated resource or media provider object has been found to be unsuitable.
+      if (this.$refs.video.error.code === 4) {
```

Comment:
Maybe we should remove this `if` and perform the node update check on any type of video loading error?

Reply:
Good catch. This solution sounds good, I can change it shortly.